content/automate/ManageIQ/Infrastructure/VM/Transform/Import.class/import_vm.yaml

```diff
@@ -0,0 +1,10 @@
+---
```

Comment:
@matobet, these two yaml files create the /ManageIQ/Infrastructure/VM/Transform/Import/ImportVm instance:
content/automate/ManageIQ/Infrastructure/VM/Transform/Import.class/importvm.yaml
content/automate/ManageIQ/Infrastructure/VM/Transform/Import.class/import_vm.yaml

You should be able to remove:
content/automate/ManageIQ/Infrastructure/VM/Transform/Import.class/import_vm.yaml
and the instance will be created properly by:
content/automate/ManageIQ/Infrastructure/VM/Transform/Import.class/importvm.yaml